cdap-ui/app/features/overview/templates/data-apps.html

```diff
@@ -21,16 +21,14 @@ <h2 class="text-uppercase"> Data </h2>
       </div>
 
       <ul class="list-group" >
-        <li class="list-group-item" ng-repeat="stream in streams track by $index">
-          <a ui-sref="streams.detail.overview({streamId: stream.name})">
-            <strong ng-bind="stream.name | caskCapitalizeFilter">  </strong>
-            <div> <p ng-bind="'Type: ' + (stream.type)"></p> </div>
+        <li class="list-group-item" ng-repeat="data in combined = streams.concat(datasets) | orderBy:'name' | limitTo: 5 track by $index ">
+          <a ng-if="data.type === 'Stream'" ui-sref="streams.detail.overview({streamId: data.name})">
+            <p ng-bind="data.name | caskCapitalizeFilter"> </p>
+            <p><img src="assets/img/stream.svg" alt="" /> Stream</p>
           </a>
-        </li>
-        <li class="list-group-item" ng-repeat="dataset in datasets track by $index">
-          <a ui-sref="datasets.detail.overview({datasetId: dataset.name})">
-            <strong ng-bind="dataset.name | caskCapitalizeFilter">  </strong>
-            <div> <p ng-bind="'Type: ' + (dataset.type)"></p> </div>
+          <a ng-if="data.type !== 'Stream'" ui-sref="datasets.detail.overview({datasetId: data.name})">
```

Comment:
Doesn't need to go in this PR but I feel this should be straight forward.

data.type === 'Dataset' & data.type === 'Stream'. Later if we have a new type of data (or icons we need to show or states we need to navigate that differs based on the type) then that should be addressed specifically for the data type.